### PR TITLE
Fix .get in python interface

### LINKED ISSUE #35: Error retrieving fields from RosValue in v0.0.38
```python
import embag

view = embag.View("my.bag")
for msg in view.getMessages(["/my/topic"]):
    rosval = msg.data()
    print(rosval.get("header"))

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Unable to convert function return value to a Python type! The signature was
	(self: embag.libembag.RosValue, arg0: str) -> Embag::RosValue::RosValuePointer
```

This happens for any field.

### PR DESCRIPTION
**Pull request by liamembark**
Description of Changes
======================
Fixes #35.

Test Plan
=========
Added test for `.get` within the existing tests.

### CODE REVIEW

File: python/embag.cc [hunk 1]
```diff
@@ -89,9 +88,8 @@ PYBIND11_MODULE(libembag, m) {
             throw std::runtime_error("Can only iterate array RosValues");
         }
       }, py::keep_alive<0, 1>() /* Essential: keep object alive while iterator exists */)
-      .def("__getattr__", [](Embag::VectorItemPointer<Embag::RosValue> &v, const std::string &attr) {
```
Comment (brendangeck) [inline python/embag.cc:92]: It almost looks like you're removing the `[]` operator here. Is this correct, or is that still supported by the `getitem` below?

File: test/embag_test_python.py [hunk 2]
```diff
@@ -97,12 +97,12 @@ def testROSMessages(self):
                     self.assertNotEqual(v, 0)
 
             if topic == '/base_pose_ground_truth':
-                self.assertEqual(msg.header.seq, base_pose_seq)
+                self.assertEqual(msg.get('header').get('seq'), base_pose_seq)
```
Comment (brendangeck) [inline test/embag_test_python.py:100]: Typically, when I see `get` in python, I expect it to function like "find a value in a dict, if that can't be found, default to some specified value or `None` (if default is not specified)". Does our code follow the same paradigm?

File: python/embag.cc (hunk 1 above)
Comment (liamembark) [reply python/embag.cc:92]: The `[]` is part of the syntax for defining a lambda. The actual `[]` operator in python is defined by `__getitem__`. The `__getattr__` interface (which provides `.` access) is still supported, it just doesn't have a lambda that wraps the `getField` and instead calls `getField` directly.

File: test/embag_test_python.py (hunk 2 above)
Comment (liamembark) [reply test/embag_test_python.py:100]: We don't support that paradigm (and we didn't before this change either). I think I'd rather leave that up to another PR to implement though, and leave this one as just a bugfix for the existing functionality that had a regression.

Comment (brendangeck) [reply test/embag_test_python.py:100]: Fair enough

**Review by brendangeck — APPROVED** [review]

**Pull request merged**